scout/index.md

```diff
@@ -51,7 +51,7 @@ Docker Scout is activated for your organization automatically.
 Advanced image analysis is a feature in Docker Hub which, when enabled, triggers
 a Docker Scout analysis every time you push an image. The analysis updates
 continuously, meaning that the vulnerability report for an image is always up to
-date as new CVEs are discovered. No need to re-scan an image.
+date as Docker Scout discovers new CVEs. No need to re-scan an image.
```

Comment:
This refers to the new CVEs becoming available in the advisories that Docker Scout integrates with, so maybe we want to phrase this differently?

Reply:
SGTM